zeebe/backup/src/test/java/io/camunda/zeebe/backup/api/BackupIdentifierWildcardTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.backup.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.zeebe.backup.api.BackupIdentifierWildcard.CheckpointPattern;
+import io.camunda.zeebe.backup.api.BackupIdentifierWildcard.CheckpointPattern.Any;
+import io.camunda.zeebe.backup.api.BackupIdentifierWildcard.CheckpointPattern.Exact;
+import io.camunda.zeebe.backup.api.BackupIdentifierWildcard.CheckpointPattern.Prefix;
+import io.camunda.zeebe.backup.common.BackupIdentifierWildcardImpl;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class BackupIdentifierWildcardTest {
+  @Test
+  void shouldBuildPrefixWithPartitionId() {
+    // given
+    final var wildcard =
+        new BackupIdentifierWildcardImpl(Optional.empty(), Optional.of(1), new Any());
```

Comment:
❓ Wondering about the ordering of the parameters. Wouldn't it make more sense to have the same order here as the output?

Like first partition id, then checkpoint and at the end the node id?